contracts/LifMarketMaker.sol

```diff
@@ -59,10 +59,11 @@ contract LifMarketMaker is Ownable {
   }
 
   function calculateDistributionPeriods(
-    uint256 startBlock, uint8 totalPeriods, uint256 blocksPerPeriod
+    uint256 _startBlock, uint8 totalPeriods, uint256 blocksPerPeriod
```

Comment:
if we use `_` to name parameters, let's do it for all of them. But also, as this function already uses (mutates) the `distributionPeriods` variable, feel free to not pass any parameters and just use the already defined MarketMaker state variables :)